### PR TITLE
react 17

### PR DESCRIPTION
**Pull request by odinr**
WIP

### CODE REVIEW

File: _babel.config.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+module.exports = {
+    // presets: [
+    //     '@babel/preset-typescript',
```
Comment (eskil0312) [inline _babel.config.js:3]: Remove config?

File: build/style.js [hunk 2]
```diff
@@ -15,15 +16,17 @@ module.exports = {
                             modules: {
                                 mode: 'local',
                                 localIdentName: 'fc--[folder]__[local]--[hash:base64:5]',
-                                context: path.resolve(__dirname, '..', 'src'),
-                                hashPrefix: new Date().getTime().toString(),
+                                // context: path.resolve(__dirname, '..', 'src'),
+                                // hashPrefix: new Date().getTime().toString(),
```
Comment (eskil0312) [inline build/style.js:20]: Remove comments?

File: .storybook/manager.js [hunk 3]
```diff
@@ -14,9 +14,14 @@ addons.setConfig({
     }),
 });
 
+/* 
 addons.register('FUSION', (api) => {
     addons.addPanel('FUSION/authtoken', {
         title: 'Auth',
-        render: ({ active, key }) => <AuthTokenInput key={key} active={active} />,
+        render: ({ active, key }) => (<AuthTokenInput key={key} active={active} />)
```
Comment (eskil0312) [inline .storybook/manager.js:21]: Remove comments?

File: .storybook/preview.js [hunk 4]
```diff
@@ -1,8 +1,8 @@
 import '../src/customElements/index.ts';
 import { addDecorator } from '@storybook/react';
-import results from '../.jest-test-results.json';
-import { withTests } from '@storybook/addon-jest';
+// // import results from '../.jest-test-results.json';
```
Comment (eskil0312) [inline .storybook/preview.js:3]: Remove comments?

**Review by eskil0312 — APPROVED** [review]

File: .storybook/preview.js (hunk 4 above)
Comment (odinr) [reply .storybook/preview.js:3]: we might reimplement jest at a later point

File: .storybook/manager.js (hunk 3 above)
Comment (odinr) [reply .storybook/manager.js:21]: the uath form does not work atm, WIP

**Pull request merged**